source/main/physics/Beam.cpp

```diff
@@ -6122,9 +6122,9 @@ bool Beam::LoadTruck(
 	}
 
 	// Calculate the approximate median
-	std::vector<Real> mx(free_node, 0.0f);
-	std::vector<Real> my(free_node, 0.0f);
-	std::vector<Real> mz(free_node, 0.0f);
+	std::vector<Ogre::Real> mx(free_node, 0.0f);
+	std::vector<Ogre::Real> my(free_node, 0.0f);
+	std::vector<Ogre::Real> mz(free_node, 0.0f);

```

Comment:
I think we should get rid of these: https://github.com/RigsOfRods/rigs-of-rods/blob/master/source/main/physics/collision/CartesianToTriangleTransform.h#L30-L32